src/Server_TEST.cc

```diff
@@ -759,6 +759,22 @@ TEST_P(ServerFixture, GetResourcePaths)
   EXPECT_EQ("/home/user/another_path", res.data(1));
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, CachedFuelWorld)
+{
+  std::string cachedWorldPath = std::string(PROJECT_SOURCE_PATH) + "/test/worlds";
+  std::string cachedWorldFilePath = cachedWorldPath +
+    "/fuel.ignitionrobotics.org/OpenRobotics/worlds/Test%20world/2/test.sdf";
```

Comment:
We should use `common::joinPaths` so that the separators are correct on Windows

Reply:
60d24050f44caa2e2abe4e52163ad600d3869b72